test/yamlOnTypeFormatting.test.ts

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { expect } from 'chai';
+import {
+  DocumentOnTypeFormattingParams,
+  FormattingOptions,
+  Position,
+  Range,
+  TextDocument,
+  TextEdit,
+} from 'vscode-languageserver';
+import { TextDocumentIdentifier } from '../src';
+import { doDocumentOnTypeFormatting } from '../src/languageservice/services/yamlOnTypeFormatting';
+
+const FILE_URI = 'file://some/file.yaml';
+function createDocument(content: string): TextDocument {
```

Comment:
I think you can probably re-use the ones from testHelper https://github.com/redhat-developer/yaml-language-server/blob/master/test/utils/testHelper.ts#L32